sphinxgallery/gen_gallery.py

```diff
@@ -1,6 +1,15 @@
 # -*- coding: utf-8 -*-
 # Author: Óscar Nájera
 # License: 3-clause BSD
+"""
+========================
+Sphinx-Gallery Generator
+========================
+
+Attaches Sphinx-Gallery to Sphinx in order to generate the galleries
```

Comment:
What about:

Loads Sphinx-Gallery into the Sphinx build chain for the example galleries to be built.